wordpress-mobile/WordPress-Android
# Add reusable Static Poster UI for Jetpack Powered Screens

Resolves partially #18115

This PR:

- ★ Implements the building blocks to enable working in parallel on the Jetpack Static Poster by adding the soon-to-be-used fragment which hosts the Compose UI barebones scaffolding for the full-screen component
- ⊕ Adds a simple system to preview fragments of to-be-implemented screens that are behind a feature flag from `Debug Settings` once the flag is toggled ON by enhancing the UI of enabled features with an actionable button that will present the specified fragment in an empty debug-only host activity that was introduced only for this use case

## To Test

1. Go to `Me` → `App Settings` → `Debug settings`
2. Scroll to see under `Remote features` `jp_removal_static_posters`
3. Tap on the item title

- **Verify** an icon appears leading the checkmark to indicate the flag is enabled
- **Verify** tapping the icon opens a rather empty fragment in a new activity
- **Verify** tapping the back button in the top-bar navigates back to debug settings
- **verify** the preview icon button is hidden when the feature is disabled

### Preview

| <video src="https://user-images.githubusercontent.com/4588074/225565817-957200c8-02d6-4771-b081-61b86f05d811.mp4" style="max-width: 315px;"></video> |
| - |

## Regression Notes
1. Potential unintended areas of impact
   None — the impacted functionality is available only in debug builds and it concerns a new feature

2. What I did to test those areas of impact (or what existing automated tests I relied on)
   N/a

3. What automated tests I added (or what prevented me from doing so)
   N/a

PR submission checklist:
